src/core/stores/VRange.ts

```diff
@@ -171,17 +258,19 @@ export class VRange {
      * @param position
      * @param reference
      */
-    setStart(reference: VNode, position = RelativePosition.BEFORE): VRange {
-        return VDocument.withRange(() => {
-            reference = reference.firstLeaf();
-            if (position === RelativePosition.AFTER && reference !== this.end) {
-                reference.after(this.start);
-            } else {
-                reference.before(this.start);
-            }
-            this._direction = null; // Invalidate range direction cache.
-            return this;
-        });
+    _setTail(reference: VNode, position = RelativePosition.BEFORE): VRange {
+        reference = reference.firstLeaf();
```

Comment:
It's funny because this commit removes the withRange that was added by the previous commit, so I guess it was not truly needed ?